test/src/unit-capi-fragment_info.cc

```diff
@@ -666,9 +671,14 @@ TEST_CASE(
   CHECK(rc == TILEDB_OK);
   CHECK(std::string(uri) == written_frag_uri);
 
-  // Get fragment name
-  const char* name;
-  rc = tiledb_fragment_info_get_fragment_name(ctx, fragment_info, 1, &name);
+  tiledb_string_t* name;
+  rc = tiledb_fragment_info_get_fragment_name_v2(ctx, fragment_info, 1, &name);
+  CHECK(rc == TILEDB_OK);
+  const char* name_ptr;
+  size_t name_length;
+  rc = tiledb_string_view(name, &name_ptr, &name_length);
+  CHECK(rc == TILEDB_OK);
+  rc = tiledb_string_free(&name);
```

Comment:
Asking again: How can this code not be copypasta?

Reply:
You mean this specific code I added or the fragment info tests in general? Should I factor the common parts of the tests as part of this PR?